<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkWidget</class>
 <widget class="QWidget" name="NetworkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Network Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBoxOperationMode">
       <property name="title">
        <string>Operation Mode</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButtonClientOperation">
          <property name="text">
           <string>Client</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonServerOperation">
          <property name="text">
           <string>Server</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxGameMode">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="title">
        <string>Game Mode</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButtonClientFirst">
          <property name="text">
           <string>Client first</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonServerFirst">
          <property name="text">
           <string>Server first</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonRandom">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEditLocalPort"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelLocalPort">
       <property name="text">
        <string>Local Port</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonConnect">
       <property name="text">
        <string>Connect to Proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStartGame">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ask Game</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
